src/main/java/seedu/condonery/model/property/Tag.java

```diff
@@ -0,0 +1,57 @@
+package seedu.condonery.model.property;
```

Comment:
doesn't seem like this file is being used! is this a duplicate of `model/tag/Tag`?

Reply:
Thanks Haoren! I wanted to delete it but forgot